unlock-app/src/components/interface/buttons/lock/Withdraw.js

```diff
@@ -2,10 +2,12 @@ import React from 'react'
 import Svg from '../../svg'
 import { LockButton } from '../Button'
 
-const Withdraw = (props) => (
-  <LockButton title='Withdraw balance' {...props}>
-    <Svg.Withdraw name="Withdraw" />
-  </LockButton>
-)
+export function Withdraw(props) {
```

Comment:
huh, why did you move back to a named function here? ES6 syntax is much cleaner?